### DISCUSSION TITLE
Help installing on windows

### DISCUSSION THREAD

**Discussion by geekygirlie** [requarks/wiki#5425] [Help / Questions]
Hi folks, I'm really struggling.  I have wiki.js running on a server but at the moment the URL is servername:port.
I want to use a DNS name and I have some already that our company no longer use so could use one of them but I'm struggling to get it working.
We use IIS and I know how to setup the DNS in there but I tried getting the site working in IIS and the DNS is working but the wiki won't render as it's showing deprecation warnings and a http status 500 error (see below).

I've tried turning off deprecation warnings and was left with the 500 error and I'm not sure why that's happening.

Or... how do I get the DNS to map to my node.js site without going through IIS?

Any ideas? Errors are below...

iisnode encountered an error when processing the request.

HRESULT: 0x2
HTTP status: 500
HTTP subStatus: 1001
HTTP reason: Internal Server Error
You are receiving this HTTP 200 response because [system.webServer/iisnode/@devErrorsEnabled](https://github.com/tjanczuk/iisnode/blob/master/src/samples/configuration/web.config) configuration setting is 'true'.

In addition to the log of stdout and stderr of the node.exe process, consider using [debugging](http://tomasz.janczuk.org/2011/11/debug-nodejs-applications-on-windows.html) and [ETW traces](http://tomasz.janczuk.org/2011/09/using-event-tracing-for-windows-to.html) to further diagnose the problem.

The last 64k of the output generated by the node.exe process to stderr is shown below:

(node:7728) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
(node:7728) [DEP0148] DeprecationWarning: Use of deprecated folder mapping "./public/" in the "exports" field module resolution of the package at F:\Wiki\node_modules\extract-files\package.json.
Update this package.json to use a subpath pattern like "./public/*".

**Comment by jorge-merino** [comment]
@geekygirlie were you able to fix this? im having a similar issue

**Reply by rrg92** [reply]
What the version of node using in iis?

Node process will start a and losten on specific port itself and serve requests. How you are configuring iis to route requests to wili?
